ls1intum/Artemis
# Temporary fix for displaying modeling exercises in exam summary

### Checklist

### Motivation and Context


Modeling exercises are displayed without any margin, which makes it hard to read.

### Description

Added a padding for apollon-editor in modeling-exam-exercise only 

### Steps for Testing


1. Create an exam with modeling-exercises
2. Participate and create models
3. In the summary page, there should be a margin around the elements in the diagram so that they are not directly at the edge and that there are no invisible elements

Known issues:
- messages on relationships will not be taken into account when calculating the size of the diagram -> they might be cut off

### Screenshots


![image (7)](https://user-images.githubusercontent.com/22955066/87712549-2311f000-c7a9-11ea-8e69-8572c00b504e.png)
![image (8)](https://user-images.githubusercontent.com/22955066/87712555-24dbb380-c7a9-11ea-81ca-ea0e0837c87b.png)
